CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,133 @@
+
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+We as members, contributors, and leaders pledge to make participation in our
+community a harassment-free experience for everyone, regardless of age, body
+size, visible or invisible disability, ethnicity, sex characteristics, gender
+identity and expression, level of experience, education, socio-economic status,
+nationality, personal appearance, race, religion, or sexual identity
+and orientation.
```

Comment:
There are deliberate line breaks throughout, is this a change made by the formatter?

If so, you can leave the code as it is now, but if not, it's better not to break lines.

Reply:
@xrxoxcxox 
The markdown format of Contribution Covenant is used as is. Therefore, the format looks like that.

The markdown format of Contribution Covenant can be found here.
https://www.contributor-covenant.org/version/2/0/code_of_conduct/code_of_conduct.md

Reply:
@ohakutsu 

I see. Then please keep the current code.